### ISSUE TITLE
[Crash]: framed drawer 2x2

### ISSUE THREAD

**Issue by Muhrks** [AllTheMods/ATM-10#793]
### Possible Fixes

Yes

### Modpack Version

10-1.0

### What happened?

Attempting to craft the Framed Drawer 2x2 from functional storage crashes the server

### Crash Log

https://gist.github.com/Muhrks/bbdc2f822640797e6ade7c8ae5945009

### Latest.Log

https://gist.github.com/Muhrks/6ce91f1efac5eec6d5874154b9be267a

**Comment by rootbear75** [comment]
we found that specifically crafting the Iron Nuggets + Drawers recipe crashes, whereas the iron nuggets + Chests does not.

**Comment by AfiliaAssela** [comment]
Yep, only Bandaid for this is removing/disabling Fast Workbench which does seem to stop the crash atleast, but the Recipe will still be not functional.

**Comment by alekthefirst** [comment]
Duplicate of https://github.com/AllTheMods/ATM-10/issues/171
Fixed in Functional Storage 1.3.4 released with ATM10 1.11
